drafts/use-cases/5.creating-new-event.md

```diff
@@ -0,0 +1,88 @@
+## Creating a new event
+
+### Story
+As an application user
+I want to be able to create new events manually
+So I can have event's I'm aware of added to my calendar.
+
+### Usage
+```javascript
+var event = {
+    "eventName": "My brand new event",
+    "eventDescription": "Hope it will work",
+    "startDate": "2017-04-19",
+    "endDate": "2017-04-19"
+};
+var client = new HydraClient();
+var operation = client.get("http://example.com")
+    .getApiDocumentation()
+    .getEntryPoint()
+    .operations
+    .filter(op => op.method == "POST" && expects == "Event")[0];
+client.invoke(operation, event);
+```
+
+### Details
+Browsing events is a must, but creating a new resource is a brand new story.
+First, the application have to be aware of how it can create a new resource.
+It could get that information either from the API documentation or entry point
+or last response's hypermedia controls. It also have to understand the type 
+of resource in scope (unless it is provided from outside of the application, 
+i.e. pass-through import from external data source).
+
+Anyway, assuming all the requirements are met, a request would look like this:
+
+```
+POST /api/events
+```
+```javascript
+{
+    "eventName": "My brand new event",
+    "eventDescription": "Hope it will work",
+    "startDate": "2017-04-19",
+    "endDate": "2017-04-19"
+}
+```
+
+```
+HTTP 201 Created
+Location: /api/events/2
+```
+
+Brand new event should be now added to the calendar.
+
+### Considerations
+
+#### Protocol agnostic specification
+While the most popular protocol in use is currently HTTP, ReST API can be implemented 
```

Comment:
I remember that in early Hydra spec there were specialized operations related to deleting or creating resources, but I think these were abandoned. Without some features available out of box this anti-pattern may be the easiest to implement.
Typed operations requires some extra implementation effort on the server side and also enforces client to understand these custom typed operations. I'm not sure it is a good direction.